1.8/administration/overlay-networks/ip-per-container.md

```diff
@@ -55,21 +53,18 @@ You can override the default network or add additional overlay networks by modif
     ssh_port: 22
     ssh_user: centos
     dcos_overlay_network:
-      vtep_subnet: 192.15.0.0/20
+      vtep_subnet: 44.128.0.0/20
       vtep_mac_oui: 70:B3:D5:00:00:00
       overlays:
+        - name: dcos
+          subnet: 9.0.0.0/8
+          prefix: 26
         - name: dcos-1
-          subnet: 192.168.0.0/17
-          prefix: 24
-        - name: dcos-2
-          subnet: 192.168.128.0/17
-          prefix: 24
-        - name: dcos-3
-          subnet: 44.128.0.0/16
+          subnet: 192.168.0.0/16
           prefix: 24
 ```
 
-In the above example, we have defined three overlay networks. The overlay networks `dcos-1` and `dcos-2` basically retain the default overlay networks, and we have added another overlay network called `dcos-3` with subnet range `44.128.0.0/16`. When you create a network, you must give it a name and a subnet. That name is used to launch Marathon tasks and other Mesos frameworks tasks using this specific overlay network.
+In the above example, we have defined two overlay networks. The overlay network `dcos` basically retains the default overlay network, and we have added another overlay network called `dcos-1` with subnet range `192.168.0.0/16`. When you create a network, you must give it a name and a subnet. That name is used to launch Marathon tasks and other Mesos framework tasks using this specific overlay network. Due to restrictions on the size of the Linux device names the overlay network name has to be less than thirteen characters. Please go through the [limitations](#limitations] section to learn more about the reasoning behind this limitation.
```

Comment:
What value does the word `basically` provide here?
As a user it makes me question what you may not be telling me here.